src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    Hello, world!
```

Comment:
Follow the indentation rules. Add an additional space of 2 spaces for nested tags.

Reply:
https://mate-academy.github.io/style-guides/htmlcss.html#indentation